src/utils/config.ts

```diff
@@ -90,6 +107,14 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   APPINSIGHTS_DISABLED: fromNullable(process.env.APPINSIGHTS_DISABLED)
     .map(_ => _.toLowerCase() === "true")
     .getOrElseL(() => true),
+  DEFAULT_TOKEN_EXPIRATION: fromNullableE(-1)(
```

Comment:
Why the default value should be set from env? We already have specific envs for L1 and L2 expirations